src/main/java/games/strategy/engine/lobby/server/db/HashedPassword.java

```diff
@@ -4,6 +4,10 @@
 
 import games.strategy.util.MD5Crypt;
 
+/**
+ * A Wrapper class for password hashes in the form of the legacy MD5 with salt system,
```

Comment:
So this javadoc should probably say this is the hashed password value we store in database, and ideally be agnostic to the hashing method (so no mention of bcrypt or md5 is needed nor really necessary)